ray-curator/ray_curator/stages/video/io/video_loading.py

```diff
@@ -29,20 +30,22 @@ def __post_init__(self):
 
     @property
     def name(self) -> str:
-        return "video_reader_download"
+        return "video_loading"
 
     def decompose(self) -> list[ProcessingStage]:
         """Decompose into constituent execution stages.
 
         Returns:
-            List of processing stages: [VideoReaderStage, VideoDownloadStage]
+            List of processing stages: [FilePartitioningStage, VideoReaderStage]
         """
-        reader_stage = VideoReaderStage(
-            input_video_path=self.input_video_path,
-            video_limit=self.video_limit
```

Comment:
Super nit should we also have video_limit be an `int | None` similar to how you have it in `FilePartitioningStage`